04-setup-prometheus-stack/README.md

```diff
@@ -79,7 +79,49 @@ Steps to follow:
 
     The chart of interest is `prometheus-community/kube-prometheus-stack`, which will install `Prometheus`, `Promtail`, `Alertmanager` and `Grafana` on the cluster. Please visit the [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack) page for more details about this chart.
 3. Then, open and inspect the `04-setup-prometheus-stack/assets/manifests/prom-stack-values-v35.5.1.yaml` file provided in the `Starter Kit` repository, using an editor of your choice (preferably with `YAML` lint support). By default, `kubeSched` and `etcd` metrics are disabled - those components are managed by `DOKS` and are not accessible to `Prometheus`. Note that `storage` is set to `emptyDir`. It means the **storage will be gone** if `Prometheus` pods restart (you will fix this later on, in the [Configuring Persistent Storage for Prometheus](#configuring-persistent-storage-for-prometheus) section).
-4. Finally, install the `kube-prometheus-stack`, using `Helm`:
+4. [OPTIONAL] If you followed - [Step 4 [OPTIONAL] - Adding an extra node for observability](../01-setup-DOKS/README.md#step-4-optional---adding-an-extra-node-for-observability) from [Chapter 1 - 01-setup-DOKS](../01-setup-DOKS/README.md) you will need to open the `04-setup-prometheus-stack/assets/manifests/prom-stack-values-v35.5.1.yaml` file provided in the `Starter Kit` repository, using a text editor of your choice (preferably with `YAML` lint support) and uncomment the `affinity` sections for both `Grafana` and `Prometheus`. The definition should look like this (for `Prometheus`):
```

Comment:
Now, this looks a little bit cumbersome for me - too many `OPTIONALs` in one sentence:
```
[OPTIONAL] If you followed - [Step 4 [OPTIONAL] ...
```

I would leave only the **first OPTIONAL**:

>4. [OPTIONAL] If you followed [Step 4 - Adding a Node Pool for the Observability Stack](../01-setup-DOKS/README.md#step-4-optional---adding-an-extra-node-for-observability) from [Chapter 1 - How to Set Up a DigitalOcean Managed Kubernetes Cluster ](../01-setup-DOKS/README.md) you will need to edit the [prom-stack-values-v35.5.1.yaml](https://github.com/digitalocean/Kubernetes-Starter-Kit-Developers/blob/main/04-setup-prometheus-stack/assets/manifests/prom-stack-values-v35.5.1.yaml) file provided in the Starter Kit repository, and uncomment the `affinity` sections for both `Grafana` and `Prometheus`:

Also, make sure to update links, references, titles where applicable.